README.md

```diff
@@ -89,6 +89,34 @@ ReactDom.render(
 );
 ```
 
+## Troubleshooting
```

Comment:
The main thought behind adding a troubleshooting section is to help when things aren't working properly, or when not connecting to a vanilla unmodified demo installation. For example when attempting to access an API that requires authenticated access to the any of the required paths the output really isn't very clear as to what went wrong.

if this isn't the appropriate place for this I'm happy to refactor/move it or raise the problems i came across as issues so that future users. I suspect that the vast majority of users will not be looking to generate a against the demo site, they will be looking to generate from their real imperfect implementations.